pkg/controllers/networking/pod_controller.go

```diff
@@ -536,6 +556,17 @@ func (r *PodReconciler) SetupWithManager(mgr ctrl.Manager) (err error) {
 		return err
 	}
 
+	// Init mac indexer for IPInstances.
+	if err = mgr.GetFieldIndexer().IndexField(context.TODO(), &networkingv1.IPInstance{}, indexerFieldMac, func(obj client.Object) []string {
```

Comment:
After the latest commit, this indexer registration may be moved to another place.